android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -793,14 +795,10 @@ private void initializePlayerSource() {
         if (source.getUri() == null) {
             return;
         }
-        DrmSessionManager drmSessionManager = initializePlayerDrm(this);
-        if (drmSessionManager == null && drmUUID != null) {
-            // Failed to intialize DRM session manager - cannot continue
-            DebugLog.e(TAG, "Failed to initialize DRM Session Manager Framework!");
-            eventEmitter.error("Failed to initialize DRM Session Manager Framework!", new Exception("DRM Session Manager Framework failure!"), "3003");
-            return;
-        }
+        /// init DRM
+        DrmSessionManager drmSessionManager = initializePlayerDrm();
```

Comment:
Above code makes breaking change.
as-is: Does not work if drmSessionManager is null.
to-be: works even if drmSessionManager is null.

We have to think about whether or not it makes sense when a user injects the right URL and a invalid(or meaningless) DRM prop.

Reply:
good catch, I restore previous code !